pom.xml

```diff
@@ -120,6 +120,11 @@
             <groupId>no.nav.common</groupId>
             <artifactId>oidc-security</artifactId>
         </dependency>
+        <dependency>
+            <groupId>no.nav.common</groupId>
+            <artifactId>oidc</artifactId>
+            <version>1.2020.03.19_11.53-26f52be04676</version>
```

Comment:
Hvorfor trenger vi denne? Hvis vi bare kopierer inn egen klasse - er denne da fremdeles relevant?

Reply:
Forstår det som du trenger den for
import no.nav.common.oidc.discovery.OidcDiscoveryConfiguration;
import no.nav.common.oidc.discovery.OidcDiscoveryConfigurationClient;

Det burde vel kunne funke - det er vel en mer isolert modul med ny pakkestruktur, så vi går ikke i beina på andre ting vi gjør.